### PR TITLE
[CI] Add node for website full build and nightly build

### PR DESCRIPTION
**Pull request by barry-jin**
## Description ##
Add linux_gpu_g4 node for website full build and nightly build

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by barry-jin** [comment]
@mxnet-bot run ci [all]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [centos-gpu, website]

**Comment by barry-jin** [comment]
@mxnet-bot run ci [unix-gpu]

File: ci/jenkins/Jenkinsfile_website_full [hunk 1]
```diff
@@ -30,7 +30,7 @@ node('restricted-utility') {
   custom_steps = load('ci/jenkins/Jenkins_steps.groovy')
 }
 
-utils.assign_node_labels(utility: 'restricted-utility', linux_cpu: 'restricted-mxnetlinux-cpu', linux_gpu: 'restricted-mxnetlinux-gpu', linux_gpu_p3: 'restricted-mxnetlinux-gpu-p3', windows_cpu: 'restricted-mxnetwindows-cpu', windows_gpu: 'restricted-mxnetwindows-gpu')
+utils.assign_node_labels(utility: 'restricted-utility', linux_cpu: 'restricted-mxnetlinux-cpu', linux_gpu: 'restricted-mxnetlinux-gpu', linux_gpu_g4: 'restricted-mxnetlinux-gpu', linux_gpu_p3: 'restricted-mxnetlinux-gpu-p3', windows_cpu: 'restricted-mxnetwindows-cpu', windows_gpu: 'restricted-mxnetwindows-gpu')
```
Comment (szha) [inline ci/jenkins/Jenkinsfile_website_full:33]: why not use linux_gpu label directly?

**Comment by szha** [comment]
@barry-jin thanks for the fix!

**Pull request merged**